File: Python/8py 1/aypb.py
"""
|--------------------------------------------------------------------------
| Are You Playing Banjo?
|--------------------------------------------------------------------------
|
"""

def are_you_playing_banjo(name):
	return f'{name} plays banjo' if name[0].lower() == 'r' else f'{name} does not play banjo'


def areYouPlayingBanjo(name):
	return name + (' plays' if name[0].lower() == 'r' else ' does not play') + " banjo"


def areYouPlayingBanjo(name):
	return f"{name} {['does not play', 'plays'][name[0] in 'Rr']} banjo"


def areYouPlayingBanjo(name):
	return "{} {} banjo".format(name, "plays" if name.lower().startswith('r') else "does not play") 


print(are_you_playing_banjo("Martin"), "martin does not play banjo");
print(are_you_playing_banjo("Rikke"), "Rikke plays banjo");
print(are_you_playing_banjo("bravo"), "bravo does not play banjo")
print(are_you_playing_banjo("rolf"), "rolf plays banjo")

File: JavaScript/8js 1/aypb.js
/*
|--------------------------------------------------------------------------
| Are You Playing Banjo?
|--------------------------------------------------------------------------
|
*/

function areYouPlayingBanjo(name) {
   return name.charAt(0).toLowerCase() == 'r' ? `${name} plays banjo` : `${name} does not play banjo`
}


function areYouPlayingBanjo(name) {
   return name + (/^r/i.test(name) ? " plays " : " does not play ") + "banjo";
}


function areYouPlayingBanjo(name) {
   return name + (name[0].toLowerCase() == 'r' ? ' plays' : ' does not play') + " banjo";
}


function areYouPlayingBanjo(name) {
   return name.match(/^r.+/i) ? `${name} plays banjo` : `${name} does not play banjo`;
}


console.log(areYouPlayingBanjo("Adam"), "Adam does not play banjo");
console.log(areYouPlayingBanjo("Paul"), "Paul does not play banjo");
console.log(areYouPlayingBanjo("Ringo"), "Ringo plays banjo");
console.log(areYouPlayingBanjo("bravo"), "bravo does not play banjo");
console.log(areYouPlayingBanjo("rolf"), "rolf plays banjo");